document-generator-api/src/main/java/uk/gov/companieshouse/document/generator/api/service/impl/DocumentGeneratorServiceImpl.java

```diff
@@ -117,12 +119,20 @@ public ResponseObject generate(DocumentRequest documentRequest, String requestId
 
         if (documentInfoResponse != null) {
             RenderDocumentResponse renderResponse = null;
+
             try {
                 renderResponse = renderSubmittedDocumentData(documentRequest, documentInfoResponse,
                         requestParameters);
-            } catch (IOException ioe) {
-                createAndLogErrorMessage("Error occurred whilst rendering the document for resource: " +
-                        requestParameters.get(RESOURCE_URI), ioe, requestParameters);
+                if (renderResponse.getStatus() >= 400) {
```

Comment:
Instead of the number 400 can we use HttpStatus.SC_BAD_REQUEST or a similar status reference for clarity.